canvas_modules/common-canvas/__tests__/test_resources/paramDefs/code_paramDef.json

```diff
@@ -48,6 +48,10 @@
       {
         "parameter_ref": "code_rows",
         "language": "text/x-rsrc",
+        "enable_maximize": true,
```

Comment:
This file should be exact copy of `harness/test_resources/parameterDefs/code_paramDef.json`.